tests/ragger/test_get_address.py

```diff
@@ -69,11 +80,65 @@ def test_get_pk(firmware: Firmware,
 
     with app_client.get_public_addr(chaincode=with_chaincode, chain_id=chain):
         navigator.navigate_and_compare(ROOT_SNAPSHOT_PATH,
-                                       "get_pk_%s" % (chain),
-                                       get_moves(firmware, navigator, chain=chain))
-    pk, addr, chaincode = ResponseParser.pk_addr(app_client.response().data, with_chaincode)
+                                       f"get_pk_{chain}",
+                                       get_moves(firmware, chain=chain))
+    pk, _, chaincode = ResponseParser.pk_addr(app_client.response().data, with_chaincode)
     ref_pk, ref_chaincode = calculate_public_key_and_chaincode(curve=CurveChoice.Secp256k1,
                                                                path="m/44'/60'/0'/0/0")
     assert pk.hex() == ref_pk
     if with_chaincode:
         assert chaincode.hex() == ref_chaincode
+
+
+def test_get_pk2(firmware: Firmware,
+                 backend: BackendInterface,
+                 navigator: Navigator):
+    app_client = EthAppClient(backend)
+
+    path="m/44'/700'/1'/0/0"
+    with app_client.get_public_addr(bip32_path=path, chaincode=True):
```

Comment:
This test should be calling the [GET_ETH2_PUBLIC_KEY](https://github.com/LedgerHQ/app-ethereum/blob/develop/doc/ethapp.adoc#get-eth2-public-key)  APDU (not yet implemented by the client)

Reply:
Done